app/db/NotificationsDB.scala

```diff
@@ -0,0 +1,38 @@
+package db
+
+import cats.implicits._
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClient
+import com.amazonaws.services.dynamodbv2.model.{AttributeValue, GetItemRequest}
+
+import config.Config
+import models.{Answer, QuestionAnswers, AtomAPIError}
+
+import scala.collection.JavaConverters._
+
+class NotificationsDB(dynamoDB: AmazonDynamoDBClient) {
+  def getNotification(atomId: String, questionId: String): Either[AtomAPIError, Option[QuestionAnswers]] = {
+    val key = Seq(
+      "atomId" -> new AttributeValue().withS(atomId), 
+      "questionId" -> new AttributeValue().withS(questionId)
+    ).toMap.asJava
+    var req = new GetItemRequest(Config.notificationsDynamoTableName, key)
```

Comment:
Does `req` need to be a `var` instead of a `val`?

Reply:
Oops, forgot to change that. Good spot!